pkg/streamer/file.go

```diff
@@ -194,128 +195,61 @@ func fileSizeUpdated(path string, latestSize int64) (int, error) {
 	}
 }
 
-// relaySubDirUpdated checks whether the relay sub directory updated
-// including file changed, created, removed, etc.
-func relaySubDirUpdated(ctx context.Context, watcherInterval time.Duration, dir string,
+// relayLogUpdatedOrNewCreated checks whether current relay log file is updated or new relay log is created.
+// we check the size of the file first, if the size is the same as the latest file, we assume there is no new wirte
```

Comment:
```suggestion
// we check the size of the file first, if the size is the same as the latest file, we assume there is no new write
```

you may need a plugin to detect typo, so many so far. you can try find rest of typos in this PR on your own before our next review 😛

Reply:
Already checked it out, thanks for your patience